limiter.go

```diff
@@ -67,7 +68,7 @@ func (l *rateLimiter) Counter() LimitCounter {
 	return l.limitCounter
 }
 
-func (l *rateLimiter) Status(key string) (bool, float64, error) {
+func (l *rateLimiter) Status(ctx context.Context, key string) (bool, float64, error) {
```

Comment:
I don't recall, what do we use the Status() method for again..? one issue here is this is an API-breaking change.. do we have to do it..?

Reply:
Ok I've changed it in a way that it's not a breaking change anymore. 

We use `Status` to gives us the `rate ( requests per defined window )`. Anyway in the end passing context is not needed, because `Status` should calculate it for the `rate limit` that was passed from config not the other limit which is only used for some specific rpc methods (MemoryStore / MemoryLoad) IMO.